src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -395,17 +395,15 @@ public synchronized void resumePool()
    // ***********************************************************************
 
    /**
-    * Log the current pool state at debug level.
+    * Log the current pool state.
     *
     * @param prefix an optional prefix to prepend the log message
     */
    void logPoolState(String... prefix)
    {
-      if (logger.isDebugEnabled()) {
-         logger.debug("{} - {}stats (total={}, active={}, idle={}, waiting={})",
-                      poolName, (prefix.length > 0 ? prefix[0] : ""),
-                      getTotalConnections(), getActiveConnections(), getIdleConnections(), getThreadsAwaitingConnection());
-      }
+      logger.info("{} - {}stats (total={}, active={}, idle={}, waiting={})",
```

Comment:
I think if really needs to be change to info, it's better to still have infoEnabled check since those info are all getting by one by one function, not simple parameter.